default/stringtables/en.txt

```diff
@@ -10511,7 +10511,7 @@ PRO_MICROGRAV_MAN
 Microgravity Industry
 
 PRO_MICROGRAV_MAN_DESC
-'''Increases [[metertype METER_INDUSTRY]] by +5 on all Industry-focused colonies in system with an asteroid belt outposts or colonies. Additional claimed asteroid belts do not provide additional benefit.
+'''Increases [[metertype METER_INDUSTRY]] by +5 on all Industry-focused colonies in system with asteroid belt outposts or colonies. Additional claimed asteroid belts do not provide additional benefit.
```

Comment:
If we're going to fix the line, let's fix it fully-- "in system" should instead be any of "in systems", "in-system", or "in a system" (my preference is "in systems").

(If I just go make that change separately it would make a conflict for your PR, so if you could please just add that here.)